packages/lib-react-components/src/Media/components/Video/Video.js

```diff
@@ -28,9 +28,7 @@ export default function Video({
       maxHeight={height}
       width='100%'
     >
-      <GrommetVideo a11yTitle={alt} controls={controlsOption} fit={fit} preload='metadata' src={src}>
-        <Anchor href={src} label={alt} />
```

Comment:
As far as I know, this nested Anchor is unused in FEM. Nesting an anchor inside a video component causes problems for Media's Video component because it cannot be wrapped in an anchor parent component (as is done in app-project's RecentSubject's components).